crates/query-engine/translation/tests/goldenfiles/sorting_by_nested_relationship_column_with_predicate/request.json

```diff
@@ -29,9 +29,7 @@
                         "name": "title",
                         "path": []
                       },
-                      "operator": {
-                        "type": "equal"
-                      },
+                      "operator": "eq",
```

Comment:
Why is this one `"eq"` but others are `"_eq"`?

Reply:
`eq` refers to the built-in equals operator from ndc-spec, `_eq` refers to the operator defined in the metadata of some tests (in tables.json). Most tests will use `eq`, but some tests might have wanted to test a metadata-defined operator? I wouldn't mind changing everything to `eq`.

Reply:
I honestly don't understand this well enough to agree or disagree, so I am going to pretend I didn't ask and trust that you know what you're doing.

Reply:
ndc-spec defines two standard operators that a connector should implement - equal (`eq`) and in (`in`). Any other operator is a custom operator defined by the connector.

So for us we implement `in` and `eq` as built-in operators that do not need to be defined, and we also allow users to define their own operators in the ndc-metadata using the `comparisonOperators` field. [Example](https://github.com/hasura/ndc-postgres/blob/268e0ca0b68f9f41316dfa50746f563924f8d7bd/crates/query-engine/translation/tests/goldenfiles/select_where_unrelated_exists/tables.json#L46).

In most tests (like this one), we use the built-in `eq` operator. In other tests where we use the `_eq` operator, we have to define it like in the example I linked.

Tests that use `_eq` also test custom comparison operators, tests that use `eq` test the built-in operator.